lib/systems/platforms.nix

```diff
@@ -28,7 +28,10 @@ rec {
 
       baseConfig = "powernv_defconfig";
       target = "vmlinux";
```

Comment:
Isn't this the thing to change?

Reply:
This needs to be changed as well but doesn't actually affect the kernel build system. The `target` field is only used by the bootloader install scripts in NixOS, as far as I can tell.

Reply:
https://github.com/NixOS/nixpkgs/blob/78f5472f4a792679879ac6a8cc0806dd1d9cac59/pkgs/os-specific/linux/kernel/manual-config.nix#L299-L305

Reply:
That convoluted chunk of code won't actually result in `kernelConf="zImage"` causing the build process to invoke `make zImage` :frown:

Reply:
> Isn't this the thing to change?

Implemented in 9298d32eab6bf636a2aa594a725effc8c0195faa

Testing now.